recipes/packages.rb

```diff
@@ -60,11 +61,27 @@
 
 # If any of our attributes are set, modify upstream packages attribute
 if packages.any? || node['osl-php']['use_ius']
-  packages <<= prefix # Prefix is also the name of the main PHP package
+  packages <<= if version.to_f < 7.3
```

Comment:
It looks like this is the case for all php 7.x IUS releases. So instead of restricting this to just 7.3, let's do the same for all 7.x IUS builds. I hadn't noticed that the mod_php7x package was also providing the php7xu package. Can you please fix this?